cumulus/polkadot-omni-node/README.md

```diff
@@ -39,12 +39,14 @@ cargo install staging-chain-spec-builder --locked
 
 ### 2. Generate a chain spec
 
-Omni Node expects for the chain spec to contain parachains related fields like `relay_chain` and `para_id`.
-These fields can be introduced by running [`staging-chain-spec-builder`](https://crates.io/crates/staging-chain-spec-builder)
-with additional flags:
+Omni Node expects for the chain spec to contain a JSON key called `relay_chain`. This is usually set
+to a chain id, mapped out by the [load_spec](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/cli/src/command.rs#L77)
+to a chain spec object corresponding to the picked relaychain.
+The field can be introduced by running [`staging-chain-spec-builder`](https://crates.io/crates/staging-chain-spec-builder)
+with a flag:
 
 ```bash
-chain-spec-builder create --relay-chain <relay_chain_id> --para-id <id> -r <runtime.wasm> named-preset <preset_name>
+chain-spec-builder create --relay-chain <relay_chain_id> -r <runtime.wasm> named-preset <preset_name>
 ```
 
```

Comment:
Maybe we should keep `para_id` in this section and just add some note about its deprecation?

It is still supported - and omni node can be used by runtime owners not wanting to migrate their runtime now. Would be good to have clear instruction how to build chain-spec with `para_id`.

I found it a  bit confusing in current shape (that's why I  commented [here](https://github.com/paritytech/polkadot-sdk/pull/8651#issuecomment-2938872190) )

Reply:
[ca97051](https://github.com/paritytech/polkadot-sdk/pull/8651/commits/ca97051cf81943aaf4786ef49978e929c63d1759)